dcos_tunnel/cli.py

```diff
@@ -33,6 +33,7 @@
                     [--host=<host>]
                     [--container=<container>]
                     [--client=<path>]
+                    [--docker=<docker-command>]
```

Comment:
Ok I'll make it `--remote-docker` for now though on first read I would assume a hostname or IP as arg to anything --remote*.